topi/python/topi/arm_cpu/conv2d.py

```diff
@@ -132,177 +143,52 @@ def _callback(op):
     traverse_inline(s, outs[0].op, _callback)
     return s
 
+@autotvm.register_topi_schedule(schedule_conv2d_nhwc, 'arm_cpu', ['direct'])
+def schedule_conv2d_nchw_arm_cpu(cfg, outs):
```

Comment:
rename to `schedule_conv2d_nhwc_arm_cpu`

Reply:
Yes, thank you!